pages/draft-06/json-schema-release-notes.md

```diff
@@ -10,41 +7,41 @@ Release notes for migrating from zyp-04 and fge-00 (draft-04) to wright-01 (draf
+| `"exclusiveMinimum"` and `"exclusiveMaximum"` | changed from a boolean to a number to be consistent with the principle of keyword independence | wherever one of these would be true before, change the value to the corresponding `"minimum"` or `"maximum"` value and remove the `"minimum"`/`"maximum"` keyword                                                                                                                                                                                                                                                                                                                                                 |
+| `"type"`                                      | definition of `"integer"`                                                                      | in draft-04, `"integer"` is listed as a primitive type and defined as "a JSON number without a fraction or exponent part"; in draft-06, `"integer"` is not considered a primitive type and is only defined in the section for keyword `"type"` as "any number with a zero fractional part"; `1.0` is thus not a valid `"integer"` type in draft-04 and earlier, but is a valid `"integer"` type in draft-06 and later; note that both drafts say that integers SHOULD be encoded in JSON without fractional parts |
+
+##### Additions and backwards-compatible changes
+
+| keyword                     | change                                                                        | consequence                                                                                                                                                               |
+| --------------------------- | ----------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| booleans as schemas         | allowable anywhere, not just `"additionalProperties"` and `"additionalItems"` | `true` is equivalent to `{}`, `false` is equivalent to `{"not": {}}`, but the intent is more clear and implementations can optimize these cases more easily               |
+| `"propertyNames"`           | added                                                                         | takes a schema which validates the _names_ of all properties rather than their values                                                                                     |
```

Comment:
todo: in the previous tables the changes are italicized, whatever the format, please ensure it's kept consistent across documents.

Reply:
Noted and applied.